inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -33,12 +33,12 @@ params:
     value: "gamma"
     choices: ["beta", "binom", "cauchy", "chisq", "exp", "f", "gamma", "geom", "hyper", "lnorm", "logis", "nbinom", "norm", "pois", "smirnov", "t", "tukey", "unif", "weibull", "wilcox"]
   data_file: 
-    label: "Name of file containing the count data over time"
-    value: "data/covid_hosp_uk_20201024.xlsx"
+    label: "Name of file containing the line list data"
```

Comment:
Here if we are implementing https://github.com/epiverse-trace/episoap/pull/137/commits/bc880aadeaeee86382ea059d65f32ea59e9a6133 , then this shouldn't be replaced but included? 
```suggestion
    label: "Name of file containing case data, whether a linelist or incidence data"
```